### PR TITLE
Japanese support

### PR DESCRIPTION
**Pull request by eponvert**
- Including Kuromoji for Japanese tokenization
- Some pre- and post-processing to get some social media conventions right
- Also, updated CHANGELOG to note Russian

### CODE REVIEW

File: src/main/scala/com/peoplepattern/text/JaLangBundle.scala [hunk 1]
```diff
@@ -0,0 +1,81 @@
...
+
+  val stopwords = LangBundle.stopwords("ja")
+
+  /**
+   * Whether the char is in the ASCII range
+   */
+  private def asciichar(c: Char): Boolean = 0.toChar <= c && c <= 255.toChar
+
+  private def asciiSplit(text: String): Vector[(String, Boolean)] = {
+    var tmp = text
```
Comment (eponvert) [inline src/main/scala/com/peoplepattern/text/JaLangBundle.scala]: I don't love the String-valued tmp, and the amount of string stuff like this below:
```scala
tmp = tmp.drop(nonascii.size)
```

**Comment by jasonbaldridge** [comment]
:+1: 

I like how you dealt with the ascii and non-ascii chars. It would make sense to abstract that for the next non-ascii language!

**Comment by eponvert** [comment]
@jasonbaldridge do you think ASCII is the right range to target? the idea is to handle URLs and hashtags etc mixed in with japanese text, but I'm not sure ASCII is right. Maybe are more general latin character range?

**Comment by jasonbaldridge** [comment]
It's a perfectly good starting point. We can update as needed if we encounter such examples.

**Comment by eponvert** [comment]
Cool, thanks!

**Pull request merged**